### ISSUE TITLE
composable-diffusion package not used on the Google Colab demo?

### ISSUE THREAD

**Issue by apolinario** [energy-based-model/Compositional-Visual-Generation-with-Composable-Diffusion-Models-PyTorch#3]
Hi, checking out the Colab demo https://colab.research.google.com/github/energy-based-model/Compositional-Visual-Generation-with-Composable-Diffusion-Models-PyTorch/blob/main/notebooks/compose_glide.ipynb 

I couldn't find where the `composable-diffusion` library is used. If I comment out the first line that installs it 
``` 
#!pip install git+https://github.com/energy-based-model/Compositional-Visual-Generation-with-Composable-Diffusion-Models-PyTorch
```
everything still works just fine

Is that the intended process? 
I thought it is likely the case that you replicate the same techniques of the paper/library in the `masks` variable and `model_fn`, `sample_64` functions and that the actual `composable-diffusion` library is not needed - but then I wondered why is it installed; would be great to get this clarified!

**Comment by nanlliu** [comment]
that can be removed for now since we haven't released some other code that we use for other models that we trained.
